galaxy/main/celerytasks/user_notifications.py

```diff
@@ -221,7 +324,35 @@ def author_release(repo_id):
 
 
 @celery.task
-def new_survey(repo_id):
+def collection_new_survey(collection_pk):
```

Comment:
I'm not getting notifications for new collection surveys.

Reply:
added hook for collection survey